### ISSUE TITLE
The browser's Back button is no longer actionable after quitting from 360 videos and resuming the app

### ISSUE THREAD

**Issue by Softvision-RemusDranca** [MozillaReality/FirefoxReality#1958]
**[Notes]:**
- If the 360 video was opened from Youtube, after resuming, the website's suggestions and comments sections are blank and the page is not scrollable. Just the previously opened video is displayed in the center of the page.
However,the website's UI is correctly displayed after a refresh.

**[Affected Versions]:**
- Firefox Reality version 5-32821846 RC1 (58b59be3)

**[Affected Platforms]:**
- Oculus Quest - Android 7.1.1
- HTC Vive Focus Plus - Android 7.1.2
- Oculus Go - Android 7.1.2

**[Prerequisites]:**
- The headset is turned on and the browser is opened.

**[Steps to reproduce]:**
1. Open and Play a 360 video in immersive mode
2. Click the "Oculus" button from the controller.
3. Click the "Quit/Home" button from the prompt displayed.
4. Resume the app and click the "Back" button.
5. Observe the behavior.

**[Expected results]:**
- The previous navigated website/page is displayed.

**[Actual results]:**
- Nothing happens.

**[Additional Notes]:**
- This issue is not reproducible on the HTC Vive Focus and Lenovo Mirage Solo devices.
- Attached is a screen recording of the issue: [link](https://drive.google.com/open?id=1ZoLXeEjZUDJ1ahf41xGpzAJCaOGFVgst)

**Comment by Softvision-RemusDranca** [comment]
After further investigation it seems that there is a work around for this issue. If a new window is opened the Back button is no longer affected for this one and the affected window can be closed.
I will remove the "Blocker" label and add the "Major" one.

**Comment by jvonitter** [comment]
This is blocked by https://bugzilla.mozilla.org/show_bug.cgi?id=1586118 and has been happening since we started restoring state on restart. Pushing to 7 to see if we can unblock it or find another solution.

**Comment by bluemarvin** [comment]
@jvonitter I've now hit this a couple of times and I don't think this is related to the restore bug. I suspect this is an FxR bug. Might be worth pulling in to v6 even if it is to root cause the issue.

**Comment by Softvision-RemusDranca** [comment]
I have verified this issue and it's no longer reproducible with the Firefox Reality 6-33100022 RC3 (1e9e4a85) build. Now the browser's Back button is actionable.
However the issue where the Youtube website's suggestions and comments sections are blank and the page is not scrollable is still reproducible. I logged a new issue for this behavior for tracking purpose (#2177).
Tested on the Oculus Quest (Android 7.1.1), Oculus Go (Android 7.1.2), HTC Vive Focus Plus (Android 7.1.2) and HTC Vive Focus (Android 7.1.2) devices.
